### PR TITLE
chore: backport and release 0.3000.9

### PR DESCRIPTION
**Pull request by c12i**
Backport #397, #398, #411 and #415 and bump dependency versions for `0.3`

### CODE REVIEW

File: templates/custom-template/custom-template/template/entry-type/ui/src/{{dna_role_name}}/{{coordinator_zome_manifest.name}}/{{#each entry_type.fields}}{{#if (and linked_from (not (eq linked_from.hash_type 'AgentPubKey') ) )}}{{kebab_case (plural ..¡entry_type.name)}}-for-{{kebab_case linked_from.name}}.ts{{¡if}}{{¡each}}.hbs [hunk 1]
```diff
@@ -26,30 +32,41 @@ export class {{pascal_case (plural ../entry_type.name)}}For{{pascal_case linked_
...
       throw new Error(`The {{camel_case linked_from.singular_arg}} property is required for the {{kebab_case (plural ../entry_type.name)}}-for-{{kebab_case linked_from.name}} element`);
     }
+
+    this.client?.on('signal', signal => {
+      if (!(SignalType.App in signal)) return;
+      if (signal.App.zome_name !== '{{../coordinator_zome_manifest.name}}') return;
+      const payload = signal.App.payload as {{pascal_case ../coordinator_zome_manifest.name}}Signal;
+      if (!(payload.type === 'EntryCreated' && payload.app_entry.type === '{{pascal_case ../entry_type.name}}')) return;
+      this._fetch{{pascal_case (plural ../entry_type.name)}}.run();
+    })
```
Comment (c12i) [inline templates/custom-template/custom-template/template/entry-type/ui/src/{{dna_role_name}}/{{coordinator_zome_manifest.name}}/{{#each entry_type.fields}}{{#if (and linked_from (not (eq linked_from.hash_type 'AgentPubKey') ) )}}{{kebab_case (plural ..¡entry_type.name)}}-for-{{kebab_case linked_from.name}}.ts{{¡if}}{{¡each}}.hbs:45]: Needs to be removed

**Comment by jost-s** [comment]
What is being backported here?

**Comment by c12i** [comment]
@jost-s updated the PR description to show the PRs I am backporting

**Comment by jost-s** [comment]
I am not equipped to do a meaningful review of this backport. I'm not familiar with the scaffolding code and there are 127 files changed with roughly 3000 lines changed, so it would be like reviewing the 4 PRs in one for me, which would take an unreasonable amount of time.

Did you cherry-pick the commits that this PR consists of?

**Comment by c12i** [comment]
@jost-s yes the commits are cherry-picked from `develop`

**Review by jost-s — APPROVED** [review]
With passing tests, that seems fine then.

**Pull request merged**